lib/demo_web/templates/user_registration/new.html.eex

```diff
@@ -0,0 +1,26 @@
+<h1>Register</h1>
+
+<%= form_for @changeset, Routes.user_registration_path(@conn, :create), fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <%= label f, :email %>
+  <%= text_input f, :email, required: true %>
+  <%= error_tag f, :email %>
+
+  <%= label f, :password %>
+  <%= password_input f, :password, required: true %>
+  <%= error_tag f, :password %>
```

Comment:
Should there be password confirmation on registration? If not, please ignore! If yes, I'll have to additionally update the corresponding changeset function.

```suggestion
  <%= error_tag f, :password %>
  
  <%= label f, :password_confirmation, "Confirm new password" %>
  <%= password_input f, :password_confirmation, required: true %>
  <%= error_tag f, :password_confirmation %>  
```

Reply:
I looked at what other apps do and it seems most of them do not require it nowadays? Maybe the goal is to reduce friction when signing up.

Reply:
fair enough!